src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/DESCryptoServiceProvider.cs

```diff
@@ -16,6 +16,7 @@ public sealed class DESCryptoServiceProvider : DES
 
         public DESCryptoServiceProvider() { }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA5351", Justification = "This is the implementation of DES")]
```

Comment:
Is it possible to put these suppressions at the class level instead of on the individual methods? (I'm actually asking; I'd prefer that in these cases, but I don't know if the analyzer infrastructure looks there.)